src/ophys_etl/modules/suite2p_wrapper/schemas.py

```diff
@@ -218,6 +226,12 @@ def check_retain_files(self, data, **kwargs):
                     'ops.npy', 'spks.npy', 'stat.npy']
         return data
 
+    @mm.post_load
+    def copy_movie_frame_rate(self, data, **kwargs):
```

Comment:
This is going to override `movie_frame_rate_hz` if both are set.

Can you modify it to throw an error if both are set. I don't want to be in a situation where users think they are getting one thing, but are actually getting another.

Unless we anticipate both being set in some circumstance....